lobbyserver.py

```diff
@@ -2002,6 +2002,11 @@ def command_hello(self, message):
                 jsonToSend = {"command": "social", "autojoin": channels}
                 self.sendJSON(jsonToSend)
 
+                # registered channels
+                query = QSqlQuery("select name from anope_db_ChannelInfo", self.parent.db)
```

Comment:
Would need to select the anope database first, or prepend it to the table name: `anope.anope_db_ChannelInfo`